docs/usage/self-hosted-experimental.md

```diff
@@ -141,6 +141,10 @@ This feature is in private beta.
 If set, Renovate will query the merge-confidence JSON API only for datasources that are part of this list.
 The expected value for this environment variable is a JSON array of strings.
 
+## `RENOVATE_X_NUGET_DISABLE_NUPKG_DOWNLOAD`
+
+If set, Renovate will not download `nupkg` files for determining package metadata.
+
```

Comment:
Didn't we agree to make this opt-in instead of opt-out?

Reply:
Also fine for me, maybe I misread the [comment above](https://github.com/renovatebot/renovate/pull/28071#discussion_r1534229452).

Reply:
Changed in https://github.com/renovatebot/renovate/pull/28071/commits/a0239dee2b1c178b6ef5b73ac2a569e11a659f09